STEER/AOD/AliAODAD.cxx

```diff
@@ -13,16 +13,12 @@
  * provided "as is" without express or implied warranty.                  *
  **************************************************************************/
 
-//-------------------------------------------------------------------------
-//     Container class for AOD AD data
-//     Author: Michal Broz
-//     michal.broz@cern.ch
-//-------------------------------------------------------------------------
-
 #include "AliAODAD.h"
 #include "AliLog.h"
 
+/// \cond CLASSIMP
```

Comment:
The `\cond` fence for `ClassImp` and `ClassDef` is no longer necessary, please remove :-)